### ISSUE TITLE
Conv2D with Torch Creates Wrong Shape

### ISSUE THREAD

**Issue by i418c** [keras-team/keras#20375]
While trying to move from Keras 2 to Keras 3 with PyTorch, I ran into a problem in my model where a Conv2D layer outputs a shape wider in one dimension and shorter in another.  It behaves appropriately when using the Tensorflow backend.

A reproducing gist is [here](https://gist.github.com/i418c/73c6e18f18eaac3fac9c9a591efca7cf).

**Comment by mehtamansi29** [comment]
Hi @i418c -

Thanks for reporting the issue. Here for torch backend running on keras3 for comparing output shape need to change kernel_size and stride like this `conv2d = Conv2D(output_size, kernel_size=(4,4),strides=(2, 2),padding='same',activation='relu', kernel_initializer='ones',use_bias=False)`. 
And also input_2d need to transpose to feed input to conv2d `inputs_2d = inputs_2d.transpose(1,2,3,0)`
              
Attached [gist](https://colab.research.google.com/gist/mehtamansi29/c048bebacfc2deb704337429144d6491/20375-conv2d-with-torch-creates-wrong-shape.ipynb) here for the reference.
